pipeline-templates/job/deploy.yml

```diff
@@ -41,6 +41,14 @@ jobs:
               LoggingRedisConnectionString: $(LoggingRedisConnectionString)
               ConfigurationStorageConnectionString: $(ConfigurationStorageConnectionString)
               SharedStorageAccountConnectionString: $(SharedStorageAccountConnectionString)
+        - template: azure-pipelines-templates/deploy/step/get-apim-subscription-key.yml@das-platform-building-blocks
+          parameters:
+            ServiceConnection: ${{ parameters.ServiceConnection }}
+            ApimResourceGroup: $(SharedApimResourceGroup)
+            ApimName: $(SharedApimName)
+            SubscriptionId: $(AppServiceName)
+            PipelineVariableName: ApprenticeshipsApimSubscriptionKey
```

Comment:
Have we asked for the environment variable defined on line 30 of [this](https://github.com/SkillsFundingAgency/das-employer-config/pull/1403/files) PR to be set to the value we're using here (`ApprenticeshipsApimSubscriptionKey`)?